src/components/Footer/Footer.module.css

```diff
@@ -11,10 +11,16 @@
 
```

Comment:
The footer height needs to be adjusted since it causes the dashboard to overflow and a user can vertically scroll the page, even though they should not be able to.

Reply:
Based on the demo yesterday, I think it looks like this was fixed in another branch and merged into dev. Correct me if I'm wrong though.

Reply:
merged.